components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Cdm/src/Storage/AdlsCsvStorage.cs

```diff
@@ -103,9 +112,9 @@ public class AdlsCsvStorage : IAdlsStorage {
                     request = _httpClient.NewRequest(
                         $"{partitionUrl}?action=append&position={contentPosition}",
                         kResource);
-                    request.SetContent(content);
+                    request.SetContent(content, UTF8Encoding.UTF8);
                     response = await _httpClient.PatchAsync(request);
-                    contentPosition += content.Length;
+                    contentPosition += UTF8Encoding.UTF8.GetByteCount(content);
```

Comment:
Just "Encoding" - no need to use the subclass here.

Reply:
done